media-fonts/polarsys-b612-fonts/polarsys-b612-fonts-1.003.ebuild

```diff
@@ -1,15 +1,15 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2021 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 inherit font vcs-snapshot
 
 DESCRIPTION="Font designed for aircraft cockpit displays"
 HOMEPAGE="https://b612-font.com/"
 SRC_URI="https://git.polarsys.org/c/b612/b612.git/snapshot/b612-bd14fde2544566e620eab106eb8d6f2b7fb1347e.tar.bz2 -> ${P}.tar.bz2"
 
-LICENSE="EPL-1.0"
+LICENSE="EPL-1.0 BSD OFL-1.1"   # to be clarified #746725
```

Comment:
Should be wrapped in "|| ( ... )"

Reply:
In the bug, ulm suggested to not wrap it. I would like to stick to his suggestion.

Reply:
Sounded to me like he doesn't care either way.

Wrapping it is correct, and I for one do actually care because OFL really isn't a free license.

Reply:
> [...] OFL really isn't a free license.

Neither the [FSF](https://www.gnu.org/licenses/license-list.html.en#SILOFL) nor the [OSI](https://opensource.org/licenses/OFL-1.1) share that opinion.